embrace-android-core/src/test/java/io/embrace/android/embracesdk/internal/config/behavior/SdkModeBehaviorImplTest.kt

```diff
@@ -2,8 +2,10 @@ package io.embrace.android.embracesdk.internal.config.behavior
 
 import io.embrace.android.embracesdk.fakes.createSdkModeBehavior
 import io.embrace.android.embracesdk.internal.config.remote.RemoteConfig
+import io.mockk.core.ValueClassSupport.boxedValue
```

Comment:
This import looks like it's not used, assuming that's the case it can be deleted